chain/network-primitives/src/types.rs

```diff
@@ -648,60 +645,6 @@ impl FromStr for PatternAddr {
     }
 }
 
```

Comment:
I'd normally agree - but at the same time, these are BorchSerialized - which means that we persist them somewhere etc.

Wondering if maybe we should keep such kinds of types in network-primitives.. (but that requires a little bit more discussion)

Reply:
@matklad @mm-near Those types shouldn't be visible to the outside world. As, they are used only by `near-network`.

Proposals:
A) Move to `peer_store.rs`
B) That being said, having a separate `db_types.rs`, which is private inside `near-network` would be an alternative option. That way, we can more easily control any changes.
C) Keep as it is.

It seems to me that `A,B > C`. I'm fine with either `A` or `B`.